stm32-data-gen/src/perimap.rs

```diff
@@ -204,6 +204,7 @@ pub static PERIMAP: RegexMap<(&str, &str, &str)> = RegexMap::new(&[
     (".*:SAI:sai1_v1_0", ("sai", "v1", "SAI")),
     (".*:SAI:sai1_v1_1", ("sai", "v2", "SAI")),
     (".*:SAI:sai1_v1_2", ("sai", "v2", "SAI")),
+    (r"STM32WB[35]5.*:SAI\d?:.*", ("sai", "v1_4pdm", "SAI")),
     (".*:SAI:sai1_v2_0", ("sai", "v1", "SAI")),
```

Comment:
Since the `RegexMap` exits at the first match, we need to match the WB55 specifically before matching `sai1_v2_0`, since that is the version assigned incorrectly to the family by CubeMX.

@Dirbaio would it be okay to depend on this implementation detail of `RegexMap`? The alternative I see is changing the DB file.

Reply:
Thanks, I'd missed that